{"name": "test_quest_can_add_book_to_basket[http://selenium1py.pythonanywhere.com/catalogue/coders-at-work_207/?promo=offer6]", "status": "skipped", "statusDetails": {"message": "XFAIL 7\n\nselenium.common.exceptions.WebDriverException: Message: Service chromedriver unexpectedly exited. Status code was: 127", "trace": "request = <SubRequest 'browser' for <Function test_quest_can_add_book_to_basket[http://selenium1py.pythonanywhere.com/catalogue/coders-at-work_207/?promo=offer6]>>\n\n    @pytest.fixture(scope='function')\n    def browser(request):\n        browser_name = request.config.getoption(\"browser_name\")\n        language = request.config.getoption(\"language\")\n        if browser_name == 'chrome':\n            options = Options()\n            options.add_experimental_option('prefs', {'intl.accept_languages': language})\n            print('\\nstart browser for test..')\n>           browser = webdriver.Chrome(options=options)\n\nconftest.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py:80: in __init__\n    super().__init__(\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py:101: in __init__\n    self.service.start()\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/common/service.py:106: in start\n    self.assert_process_still_running()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.chrome.service.Service object at 0x7f4652a4c2d0>\n\n    def assert_process_still_running(self) -> None:\n        \"\"\"Check if the underlying process is still running.\"\"\"\n        return_code = self.process.poll()\n        if return_code:\n>           raise WebDriverException(f\"Service {self.path} unexpectedly exited. Status code was: {return_code}\")\nE           selenium.common.exceptions.WebDriverException: Message: Service chromedriver unexpectedly exited. Status code was: 127\n\n/usr/local/lib/python3.11/site-packages/selenium/webdriver/common/service.py:119: WebDriverException"}, "parameters": [{"name": "link", "value": "'http://selenium1py.pythonanywhere.com/catalogue/coders-at-work_207/?promo=offer6'"}], "start": 1681420804264, "stop": 1681420804264, "uuid": "63652a76-c9f3-4d20-9ee9-88a2925deb0b", "historyId": "4cfcafafcb0c2eb155581691b4fd5459", "testCaseId": "a51f5430e162d77677d100673d264894", "fullName": "tests.test_product_page#test_quest_can_add_book_to_basket", "labels": [{"name": "epic", "value": "test of don't remember"}, {"name": "tag", "value": "need_review"}, {"name": "tag", "value": "@pytest.mark.xfail(reason='7')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_product_page"}, {"name": "host", "value": "de6288d1d106"}, {"name": "thread", "value": "8-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_product_page"}]}